jaeger-core/README.md

```diff
@@ -52,6 +56,14 @@ JAEGER_SAMPLER_MANAGER_HOST_PORT | no | The host name and port when using the re
 JAEGER_TAGS | no | A comma separated list of `name = value` tracer level tags, which get added to all reported spans. The value can also refer to an environment variable using the format `${envVarName:default}`, where the `:default` is optional, and identifies a value to be used if the environment variable cannot be found
 JAEGER_DISABLE_GLOBAL_TRACER | no | If set, disables registering the tracer with io.opentracing.GlobalTracer
 
+Setting `JAEGER_ENDPOINT` causes the client to use the `HttpSender` to submit traces to a remote
+endpoint, usually served by a Jaeger Collector. If the endpoint is secured, an HTTP Basic Authentication
+can be performed by setting the related environment vars. Similarly, if the endpoint expects an authentication
+token, like a JWT, set the `JAEGER_AUTH_TOKEN` environment variable. If the Basic Authentication environment
```

Comment:
I would split the text for authentication from the connection properties. So deal with endpoint vs agent host/port first - and then as part of the authentication para can say only supports when using the 'ENDPOINT' property.